src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,7 +14,10 @@
     private Phone phone;
     private Email email;
     private Address address;
-
+    private int sequenceNumber;
+    public static int nextSequenceNumber = 1;
```

Comment:
`nextSequenceNumber` does not need to be public to be used in Person class' constructor.